nginx/nginx.conf

```diff
@@ -12,8 +12,8 @@ http {
          server {
             listen 80;
             listen 443 ssl;
-            ssl_certificate /etc/ssl/example.com+5.pem;
-            ssl_certificate_key /etc/ssl/example.com+5-key.pem;
+            ssl_certificate /etc/ssl/ssl.pem;
+            ssl_certificate_key /etc/ssl/ssl.key;
```

Comment:
`example.com+5.pem` and `example.com+5-key.pem` should be renamed for dev deployments to work.

Reply:
Dev deployments have their own conf file: https://github.com/cznethub/dsp/blob/develop/nginx/nginx-dev.conf